lib/bibliothecary/parsers/conda.rb

```diff
@@ -10,18 +10,22 @@ def self.mapping
         {
           match_filename("environment.yml") => {
             parser: :parse_conda_manifest,
-            kind: "manifest"
+            kind: %w[manifest lockfile]
```

Comment:
Should be just one entry here for "environment.yaml"? I'm confused why it's duplicated here. I would think it would always just match on the first entry.

Reply:
YML vs YAML

Reply:
Ah got it 👍